Documentation/contributing/development/contributing_guide.rst

```diff
@@ -127,36 +143,48 @@ requirements have been met:
       item to the first line.
 
-#. Pick the right labels for your PR:
+#. If you have permissions, pick the right labels for your PR:
 
    +------------------------------+---------------------------------------------------------------------------+
    | Labels                       | When to set                                                               |
    +==============================+===========================================================================+
    | ``kind/bug``                 | This is a bugfix worth mentioning in the release notes                    |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``kind/enhancement``         | This is an enhancement/feature                                            |
+   | ``kind/enhancement``         | This enhances existing functionality in Cilium                            |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``priority/release-blocker`` | This PR should block the current release                                  |
+   | ``kind/feature``             | This is an feature                                                        |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``area/*``                   | Code area this PR covers                                                  |
+   | ``priority/release-blocker`` | This PR should block the current release                                  |
    +------------------------------+---------------------------------------------------------------------------+
    | ``needs-backport/X.Y``       | PR needs to be backported to these stable releases                        |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``pending-review``           | PR is immediately ready for review                                        |
-   +------------------------------+---------------------------------------------------------------------------+
-   | ``wip``                      | PR is still work in progress, signals reviewers to hold.                  |
-   +------------------------------+---------------------------------------------------------------------------+
    | ``backport/X.Y``             | This is backport PR, may only be set as part of :ref:`backport_process`   |
    +------------------------------+---------------------------------------------------------------------------+
    | ``upgrade-impact``           | The code changes have a potential upgrade impact                          |
    +------------------------------+---------------------------------------------------------------------------+
+   | ``area/*`` (Optional)        | Code area this PR covers                                                  |
```

Comment:
Might be useful to clarify the use case(s) for these labels if we want folks to use them. I'm guessing they're only used to search/triage issues and pull requests?

Reply:
I was planning to remove this line and then I decided to add the `(Optional)` here.